tests/aequilibrae/paths/test_traffic_assignment.py

```diff
@@ -262,8 +263,41 @@ def test_execute_and_save_results(
         with pytest.raises(ValueError):
             assignment.save_results("save_to_database")
 
-    def test_execute_no_project(self, project: Project, assignment: TrafficAssignment, assigclass: TrafficClass):
-        conn = sqlite3.connect(os.path.join(siouxfalls_project, "project_database.sqlite"))
+        # Let's test logging of assignment
+        log_ = Path(proj.path_to_file).parent / "aequilibrae.log"
+        assert isfile(log_)
+
+        with open(log_, encoding="utf-8") as file:
+            # lines = ";".join([x.rstrip() for x in file.readlines()])
+            lines = [x.rstrip() for x in file.readlines()]
+
+        assert lines[8].split(";", 1)[1] == lines[27].split(";", 1)[1] == lines[536].split(";", 1)[1] == \
+               lines[658].split(";", 1)[1] == lines[713].split(";", 1)[1]
+
+        tclass_data = lines[8].split(";")[2].replace("\\", "").replace("'", "").split(", ")
+        assert tclass_data[:5] == [' {car: {Graph: "{Mode: c',
+                                   "Block through centroids: False",
+                                   "Number of centroids: 24",
+                                   "Links: 76",
+                                   'Nodes: 24}"',
+                                   ]
+        assert tclass_data[-2:] == ["Matrix cores: [matrix]", 'Matrix totals: {matrix: 360600.0}}"}}']
+
+        assig_data_1 = lines[10].split(";")[2].replace("\\", "").replace("'", "").split(", ")
```

Comment:
By targeting specific lines of the log file, we are putting ourselves in a situation where any future change to logging will break this test.  We can, instead of doing this, merge all lines into a single string and check if entire sections (line the one on line 28 is included in the total of the logging